### DISCUSSION TITLE
Phalcon Session cookie supprted Samesite?

### DISCUSSION THREAD

**Discussion by AliShareei** [phalcon/phalcon#97] [Q&A]
Hi 

As you know, some browsers like Chrome have given more importance to the SameSite Attribute and their behavior has changed recently. 

in Phalcon Added this feature for Cookies Phalcon/cphalcon#14683 
But I do not know if this problem is solved for session cookies or not? 

Example related link
https://stackoverflow.com/questions/64442252/session-lose-on-bank-redirect-phalconphp

**Comment by Jeckerson** [comment]
Hello, inside PR that you specified are tests, from which you can pick the code and test inside your application.

For example:
```php
$name     = 'test';
$value    = "phalcon";
$expire   = time() - 100;
$path     = "/";
$secure   = true;
$domain   = "phalcon.ld";
$httpOnly = true;
$options  = ["samesite" => "Lax"];

$cookie = new Cookie(
    $name,
    $value,
    $expire,
    $path,
    $secure,
    $domain,
    $httpOnly,
    $options
);
```

**Reply by AliShareei** [reply]
Thanks but this is for Cookies. What about sessions? 
The problem is session key stored by cookie on browser and I mean that cookie.

in pure PHP can set by `session_set_cookie_params` before session_start

```
  session_set_cookie_params([
      'path' => $path,
      'httponly' => true,
      'samesite' => $sameSite,
      'secure' => $secure,
  ]);
```

**Reply by AliShareei** [reply]
@Jeckerson , You think, Should I create an `issue` on `phalcon/cphalcon` for this, or just am I wrong?
